distributions/LibreELEC/show_config

```diff
@@ -0,0 +1,46 @@
+show_distro_config() {
+  # Hardware decoder support
+
+  config_message+="\n\n Kodi Hardware configuration:"
+  config_message+="\n $dashes$dashes"
+
+  config_message+="\n - Video Acceleration provider:\t\t $KODIPLAYER_DRIVER"
+
+  # Input device configuration
+
+#  config_message+="\n\n Input device configuration:"
+#  config_message+="\n $dashes$dashes"
```

Comment:
intentional out-commented ?

Reply:
I was on the fence about this. It's the sub-header for remotes, CEC settings, and joysticks, which currently exists. I merged this content with Kodi's video acceleration provider, which came out of the Graphics configuration, and I thought it could all live under a "kodi hardware config" header, but thought others should weigh in.

If happy as is, it should be deleted.

Reply:
I don't care, maybe others ? if nobody complians just remove it, could be added later again if someone needs it.

Reply:
Lines removed.